src/toxcore/messenger/packet/mod.rs

```diff
@@ -29,7 +29,7 @@ pub use self::user_status::*;
 pub use self::file_data::*;
 pub use self::status_message::*;
 pub use self::file_send_request::*;
-pub use self::group::*;
+pub use self::group::Packet as GroupPacket;
```

Comment:
This way you expose only `GroupPacket`. You should add * for all other packets.

Reply:
>pub use self::group::{Packet as GroupPacket, *};

can't compile

```
   Compiling tox v0.0.9 (/Users/namsoo/work/tox)
error[E0599]: no variant named `Online` found for type `toxcore::messenger::packet::group::Packet` in the current scope
   --> src/toxcore/messenger/packet/mod.rs:106:17
    |
106 |         Packet::Online(Online)
    |         --------^^^^^^
    |         |
    |         variant not found in `toxcore::messenger::packet::group::Packet`
    | 
   ::: src/toxcore/messenger/packet/group/mod.rs:104:1
    |
104 | pub enum Packet {
    | --------------- variant `Online` not found here
```

Reply:
Then we probably shoudn't use `pub use`, but `pub mod` instead. We use `pub use` for single packets but `group` module defines a whole bunch of packets.

Reply:
I changed structure of directory.